v2/decoder_account.go

```diff
@@ -41,13 +41,12 @@ func loadAccount(data []byte, version int) (*AccountClaims, error) {
 		return v1a.Migrate()
 	case 2:
 		var v2a AccountClaims
-		v2a.Limits.HAResources = NoLimit
 		v2a.SigningKeys = make(SigningKeys)
 		if err := json.Unmarshal(data, &v2a); err != nil {
 			return nil, err
 		}
-		if !v2a.Limits.IsJSEnabled() {
-			v2a.Limits.HAResources = 0
+		if len(v2a.Limits.TieredLimits) > 0 {
+			v2a.Limits.JetStreamLimits = JetStreamLimits{}
```

Comment:
So again, here it means that v2a.Limits.JetStreamLimits would be empty, which means that if IsJSEnabled() was ran against v2a.Limits.JetStreamLimits(), the result would be false. Is that an issue?